stardist/models/model2d.py

```diff
@@ -512,6 +512,10 @@ def _instances_from_prediction(self, img_shape, prob, dist, points=None, prob_cl
                 inds = tuple(p//g for p,g in zip(points.T, self.config.grid))
                 prob_class = prob_class[inds]
 
+        if scale is not None:
+            points = points/scale
+            disti = disti/scale
+
         if return_labels:
             labels = polygons_to_label(disti, points, prob=probi, shape=img_shape)
```

Comment:
Can't do multiple scale values (one per axis) because `polygons_to_label` uses distances, and not coordinates.

Reply:
We could easily update `polygons_to_label` to also take `scale` as an input, but it's not so easy in 3D, where the C++ code needs to be updated.

Reply:
I think that makes sense, because a one might not want to scale the `Z` axis in the same way as `X` and `Y`.

Reply:
I've already implemented per axis scale values in `predict_instances` and could pass a dict (axis -> scale) on to `__instances_from_prediction`.

Reply:
I could modify the 3D C++ code, but I might screw it up somewhere ;)

Reply:
sounds good!

Reply:
> I could modify the 3D C++ code, bu

Yes, please don't :)